### ISSUE TITLE
Issue: IndexError: index out of range in self

### ISSUE THREAD

**Issue by alxhrzg** [mickeystroller/R-BERT#5]
Seems to be related to the added special tokens, can be fixed by adding: 
model.resize_token_embeddings(len(tokenizer))
See a related issue with BERT: https://github.com/huggingface/transformers/issues/4153

**Comment by NONOThingC** [comment]
This is aso helpful for 
`IndexError: index out of range in self`
But it is strange that training in gpu seems dont have this problem.

**Comment by alxhrzg** [comment]
Yes indeed, I opened this issue in a hurry (before I forget) and actually meant _IndexError: index out of range in self_. I will adjust it accordingly!
